app/utils/add_sheets_items.py

```diff
@@ -11,314 +10,132 @@
 )
 
 # data sheets objects. bot_sheet -> already existing bot data andela_sheet -> Andela equip data sheet
-andela_sheet = gc.open("Asset Tracker For bot").worksheet("Master  Inventory List")
-
+andela_sheet = gc.open('Benjamin Copy of Asset Tracker For bot').worksheet("Master  Inventory List")
 
-def get_serial_values(sheet, col, col_value, col_num=1):
+def get_all_items(sheet):
     """
-    Function to return the cell range of asset_code or serial number of an item
-    in a sheet.
-
-    Example: Get all cells with asset-code for each thunderbolt
+    Function to return a list of all items in the master sheet
+    Example:
+        get_all_items('Master Sheet')
 
-    :param sheet: sheet the tb data is in
-    :param col: column which has serial number or asset_code
-    :param col_value: The label of the field which says an item is a tb or mac
-    :param col_num: column number of the column with the data
-    :rtype: list
-    :return: serial_nums - list of cell objects in the range
+    :param sheet: The name of the sheet to use e.g 'Andela Asset Tracker'
     """
-    logging.info('Getting all cells under asset_code column')
-    item_cells = []
-    and_items = sheet.findall(col_value)
-    for item in and_items:
-        item_cells.append(item.row)
 
-    # list of lists of consecutive numbers
-    cell_lists = [list(map(itemgetter(1), g)) for k, g
-                  in groupby(enumerate(item_cells), lambda x: x[0] - x[1])]
-    cell_ranges = []
-    for cell_list in cell_lists:
-        cell_range = '{}{}:{}{}'.format(col, cell_list[0], col, cell_list[-1])
-        cell_ranges.extend(sheet.range(cell_range))
+     # Get and write thunderbolts
+    and_sheet = gc.open(sheet).worksheet("Master  Inventory List")
 
-    logging.info('{} cells found in sheet {}worksheet'.format(
-        len(cell_ranges), sheet.title))
+    # get a list of all assets in the andela_sheet
+    and_items = and_sheet.get_all_values()
 
-    return cell_ranges
+    data = {
+            'macbooks': macbooks(and_items),
+            'thunderbolts': thunderbolts(and_items),
+            'chargers': mac_chargers(and_items),
+            'dongles': []
+    }
 
+    # create the equipments.json file
+    json_file = open("equipment.json", "a+")
```

Comment:
You're appending to the file. Any reason why we're not flushing everything out before writing?